plugins/Toolbox/src/Toolbox.py

```diff
@@ -31,8 +31,8 @@
 
 ##  The Toolbox class is responsible of communicating with the server through the API
 class Toolbox(QObject, Extension):
-    DEFAULT_CLOUD_API_ROOT = "https://api.ultimaker.com" #type: str
-    DEFAULT_CLOUD_API_VERSION = 1 #type: int
+    DEFAULT_CLOUD_API_ROOT = "https://api.ultimaker.com"  # type: str
```

Comment:
This could now be fetched from the CuraConstants that @LipuFei is working on: https://github.com/Ultimaker/Cura/commit/8e4ad23da104ead8ee5ce3658e7132d3d8e9eaec.